modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -457,6 +468,39 @@ public void testAddObjectAction() throws Exception {
 
 			Assert.assertEquals(0, _argumentsList.size());
 
+			_objectEntryLocalService.addObjectEntry(
+				TestPropsValues.getUserId(), 0,
+				_objectDefinition.getObjectDefinitionId(),
+				HashMapBuilder.<String, Serializable>put(
+					"firstName", "John"
+				).put(
+					"lastName", "Smith"
+				).build(),
+				ServiceContextTestUtil.getServiceContext());
+
+			// Then actions must not be triggered in an infinite loop
```

Comment:
@devpedrotavares please change to `Related object actions must be executed correctly to avoid infinite loop`